lib/asset_sync/storage.rb

```diff
@@ -58,7 +58,7 @@ def files_to_invalidate
       self.config.invalidate.map { |filename| File.join("/", self.config.assets_prefix, filename) }
     end
 
-    def get_local_files
+    def get_asset_files_from_manifest
```

Comment:
Add a code comment indicating this is a public API for other gem(s)

Reply:
I did it!